recipes/epysurv/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "epysurv" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  url: https://github.com/jarnorfb/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 56383ce024e2ad76c1ac300065efe555cc57fcbc8d7ab00052b310195470e80c
+
+build:
+  # noarch: python
+  number: 0
+  merge_build_host: True  # [win]
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    # if your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('cxx') }}
+  host:
+    - python=3.7
+    - pip
+    - r-base=3.5
+    - rpy2
+    - numpy=1.16
+    - r-essentials
```

Comment:
r-essentials is a wrapper recipe for many R packages ([full list](https://github.com/conda-forge/r-essentials-feedstock/blob/master/recipe/meta.yaml#L17)). Please only list R packages that are required for your software to run.

Reply:
@jdblischak Sorry for being unresponsive. I was quite busy with other stuff recently. Having an r-surveillance package is really nice.  I will now try to adopt the build according to your comments.

Reply:
@JarnoRFB No worries! I totally understand